lms/templates/certificates/_accomplishment-footer.html

```diff
@@ -1,16 +1,17 @@
+<%page expression_filter="h"/>
 <div class="wrapper-footer">
 
     <footer class="footer-app" role="contentinfo" id="company-info">
         <div class="footer-app-copyright">
-            <p class="copy copy-micro">${copyright_text}</p>
+            <p class="copy copy-micro">${copyright_text | n, decode.utf8 }</p>
         </div>
         <nav class="footer-app-nav">
             <ul class="list list-legal">
                 <li class="nav-item">
-                    <a class="action btn btn-small btn-link" href="${company_tos_url}">${company_tos_urltext}</a>
+                    <a class="action btn btn-small btn-link" href="${company_tos_url | n, decode.utf8 }">${company_tos_urltext}</a>
                 </li>
                 <li class="nav-item">
-                    <a class="action btn btn-small btn-link" href="${company_privacy_url}">${company_privacy_urltext}</a>
+                    <a class="action btn btn-small btn-link" href="${company_privacy_url | n, decode.utf8 }">${company_privacy_urltext}</a>
```

Comment:
We still need to apply HTML escaping to these URLs. Please remove the n filter. See http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/preventing_xss.html#url-context-and-escaping for more details.

Reply:
I'll check this document to see how i can add html escaping to all url.

Reply:
If you remove the n filter, the default HTML escaping logic should be applied.